models/ios/logging_global/ios_logging_global.yaml

```
@@ -0,0 +1,1071 @@
+            type: dict
+            suboptions:
+              level:           
+                description: Logging severity level (0-7)
+                type: int
+                choices: [0,1,2,3,4,5,6,7]
+              severity:
+                description: Logging severity level
+                type: str
+                choices: ["alerts","critical","debugging","emergencies","errors","informational","notifications","warnings"]
+      count:                
+        description: Count every log message and timestamp last occurance
+        type: bool
+      delimiter:    
+        description: Append delimiter to syslog messages
+        type: str
+        suboptions:
+          tcp:           
+            description: Append delimiter to syslog messages over TCP
+            type: bool
+      discriminator:
+        description: Create or modify a message discriminator
+        type: str
+      dmvpn:          
+        description: DMVPN Configuration
+        type: dict
+        suboptions:
+          rate_limit:           
+            description: rate in messages/minute, default is 600 messages/minute (1-10000)
+            type: int
+      esm:        
+        description: Set ESM filter restrictions
+        type: str
+        suboptions:
+          config:           
+            description: Permit/Deny configuration changes from ESM filters
+            type: bool
+      exception:    
+        description: Limit size of exception flush output (4096-2147483647)
+        type: int
+      facility:     
+        description: Facility parameter for syslog messages
+        type: str
+        choices: ["auth","cron","daemon","kern","local0","local1","local2","local3","local4","local5","local6","local7","lpr","mail","news","sys10","sys11","sys12","sys13","sys14","sys9","syslog","user","uucp"]
+      filter:     
+        description: Specify logging filter
+        type: dict
+        suboptions:
+          url: 
+            description: Filter Uniform Resource Locator
+            type: str
+          order: 
+            description: Order of filter execution
+            type: int
+          args:
+            description: Arguments passed to filter module.
+            type: str
+      history:
+        description: Configure syslog history table
+        type: dict
+        suboptions:
+          level:           
+            description: Logging severity level (0-7)
+            type: int
+            choices: [0,1,2,3,4,5,6,7]
+          size:            
+            description: Set history table size (0-500)
+            type: int
+          severity:
+            description: Logging severity level
+            type: str
+            choices: ["alerts","critical","debugging","emergencies","errors","informational","notifications","warnings"]
+      host:           
+        description: Set syslog server IP address and parameters
+        type: dict
```

host can have multiple values correct, if yes then it should be list of dict instead of dict.
```suggestion
        type: list
        elements: dict
```
Please verify the model against the other valid list of dict params